soccer/GameSettings.hpp

```diff
@@ -0,0 +1,40 @@
+struct GameSettings {
+    
+    bool is_simulation;
+
+    bool useOurHalf, useOpponentHalf;
+
+    // True if we are running with a simulator.
+    // This changes network communications.
+    bool simulation;
+
+    // True if we are blue.
+    // False if we are yellow.
+    bool blueTeam;
+
+    // _teamTrans is used for positions, not angles.
+    // _teamAngle is used for angles.
+    float teamAngle;
+
+    // Board ID of the robot to manually control or -1 if none
+    int manualID;
```

Comment:
As far as I remember multiple manual is broken. Even if it's not it should get an overhaul. Regardless, this is what it was like previously, so I think fixing multiple manual stuff is beyond the scope of this PR.

Reply:
I was moreso making sure that this wasn't a straight duplicate of what's in context already

Reply:
Oh, right. https://github.com/RoboJackets/robocup-software/blob/staging/soccer/Context.hpp#L34.

I think probably the best thing to do would be to move the `is_joystick_controlled` array into `GameSettings` and then set it in `MainWindow` (or maybe make it a `std::array<int, 16>` where it's -1 for not joystick controlled and the joystick ID otherwise). That would mean we could get rid of all of the multiple manual code and deal with that later.

Reply:
@HussainGynai ping on this. Again, I think the best thing to do would be a `std::array<int, 16> robot_joystick_ids` in `GameSettings`, with `-1` corresponding to "no joystick controller".